### PR TITLE
Bubblesort R implementation

### PR DESCRIPTION
**Pull request by tsik1**

### CODE REVIEW

File: contents/bubble_sort/code/r/bubblesort.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
+bubble_sort <- function(array) {
+  len_array <- length(array)
+  for (i in 1:(len_array-1)) {
+    for (j in 1:(len_array-i)) {
+      if (array[j] > array[j+1]) {
+        #'temp' will hold the current array[j] value because it will be changed
+        temp <- array[j]
+        array[j] <- array[j+1]
+        array[j+1] <- temp
```
Comment (Trashtalk217) [inline contents/bubble_sort/code/r/bubblesort.R]: Please make sure to place spaces around operators ('+', '-' and '*') to improve readability. See [here](https://google.github.io/styleguide/Rguide.xml)

File: contents/bubble_sort/code/r/bubblesort.R [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      }
+    }
+  }
+  return(array)
+}
+
+main_function <- function() {
+  #the sample function creates a vector/1d array with random integers
+  #100 states the right endpoint of the closed interval, [1, 100]
+  #15 states the amount of integers we want in our vector/1d array
```
Comment (Trashtalk217) [inline contents/bubble_sort/code/r/bubblesort.R]: Take another look at your comments and add spaces between the hashtag and the sentence.

File: contents/bubble_sort/code/r/bubblesort.R [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  random_array <- sample(100, 15)
+  print("Before sorting:")
+  print(random_array)
+  print("After sorting:")
+  print(bubble_sort(random_array))
+}
+
+main_function()
+
+
```
Comment (Trashtalk217) [inline contents/bubble_sort/code/r/bubblesort.R]: There's an extra line too many. Usually there is one extra line after the last text, but here there are two.

File: contents/bubble_sort/bubble_sort.md [hunk 4]
```diff
@@ -141,6 +143,8 @@ Trust me, there are plenty of more complicated algorithms that do precisely the
 [import, lang:"emojicode"](code/emojicode/bubble_sort.emojic)
 {% sample lang="bash" %}
 [import, lang:"bash"](code/bash/bubble_sort.bash)
+{% sample lang="r" %}
+[import, lang:"r"](code/r/bubblesort.r)
```
Comment (Trashtalk217) [inline contents/bubble_sort/bubble_sort.md]: Use '.R' instead of '.r' otherwise the build fails.

**Comment by c252** [comment]
Hey @tsik1 , just checking in on these R PRs? Are you planning to work on them again? I am only asking because we have a duplicate PR #636 .

**Comment by leios** [comment]
Already completed in #655 sorry for the trouble!

**Pull request closed without merging**